Code/Source/URDF/FbxNode.cpp

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "FbxNode.h"
+
+#include <fstream>
+#include <string>
+
+#include <AzCore/Debug/Trace.h>
+
+namespace ROS2
+{
+    namespace Fbx
+    {
+        std::string AnyToString(const std::any & a)
+        {
+            if (a.type() == typeid(int))
+            {
+                return std::to_string(std::any_cast<int>(a));
+            }
+            else if (a.type() == typeid(unsigned))
+            {
+                return std::to_string(std::any_cast<unsigned>(a));
+            }
+            else if (a.type() == typeid(float))
+            {
+                return std::to_string(std::any_cast<float>(a));
+            }
+            else if (a.type() == typeid(double))
+            {
+                return std::to_string(std::any_cast<double>(a));
+            }
+            else if (a.type() == typeid(const char*))
+            {
+                return std::string("\"") + std::any_cast<const char*>(a) + std::string("\"");
+            }
+            else if (a.type() == typeid(std::string))
+            {
+                return std::string("\"") + std::any_cast<std::string>(a) + std::string("\"");
+            }
+            else if (a.type() == typeid(RawString))
+            {
+                return std::any_cast<RawString>(a).data;
+            }
+
```

Comment:
Should put an assert here or something to catch any type that is not being handled, or at least a warning